utils/rpms/daos.spec

```diff
@@ -168,18 +170,23 @@ Requires: mercury >= %{mercury_version}
 Requires: libfabric >= %{libfabric_version}, libfabric < %{libfabric_max_version}
 %if (0%{?rhel} >= 8)
 Requires: fuse3 >= 3
+Requires: capstone >= 4.0.1
 %else
 Requires: fuse3 >= 3.4.2
+Requires: capstone >= 4.0.1
 %endif
```

Comment:
This all appears to be a duplication of https://github.com/daos-stack/daos/blob/fbf14cdfc09f7e638dbacd772edf04bd30e42656/utils/rpms/daos.spec#L184-L190 and so likely should just be removed:

```suggestion
```

Reply:
I removed capstone. Thank you!